src/main/java/org/cyclops/cyclopscore/config/configurabletypeaction/MobAction.java

```diff
@@ -14,21 +13,22 @@
  * @author rubensworks
  * @see ConfigurableTypeAction
  */
-public class MobAction<T extends Entity> extends ConfigurableTypeAction<MobConfig<T>>{
+public class MobAction<T extends EntityLiving> extends ConfigurableTypeAction<MobConfig<T>>{
```

Comment:
This should actually be `EntityLivingBase`, its parent class. I see the class cast (line 30) used to be wrong then as well, could you update this in this PR as well?